index.html

```diff
@@ -1,51 +1,12 @@
-<!DOCTYPE html>
-<meta charset="utf-8">
+<!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Engine and Standard libs benchmarks</title>
-  <style>
-    body {
-      font-family: Arial, sans-serif;
-      font-size: 25px;
-      text-align: inherit;
-      display: flex;
-      justify-content: center;
-      align-items: center;
-      flex-direction: column;
-      min-height: 100vh;
-      background-color: #f4f4f4;
-    }
-    .container {
-      width: 90%;
-      max-width: 1200px;
-      background-color: #fff;
-      padding: 40px;
-      box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
-    }
-    h1 {
-      text-align: center;
-      font-size: 2.5em;
-      color: #333;
-    }
-    ul {
-      list-style-type: none;
-    }
-    li {
-      padding-top: 10px;
-      padding-bottom: 10px;
-      text-align: center;
-    }
-  </style>
-</head>
-<body>
-  <div class="container">
-    <h1>Engine and Standard libs benchmarks</h1>
-    <ul>
-      <li><a href="engine-benchs.html">Engine Benchmarks</a></li>
```

Comment:
These old links should be kept until we decide the new infrastructure is a complete and good enough replacement for the old one. You can move the `<div class="container">` to the bottom of the `index.html` - it doesn't have to take a prominent place.

Reply:
The static websites are reintroduced in 4e9f6bf8928a721a590347bb14f07b19694d84e7 from the previous [merge with main](c61396ec0c314b16b13e96dbbad46fa3a3ca4635).